src/praisonai/praisonai/ui/_aiui_datastore.py

```
@@ -11,30 +11,40 @@
+    def __init__(self, store: Optional[Any] = None):
+        """Initialize with an optional session store, defaults to hierarchical store.
+        
+        Args:
+            store: Optional SessionStoreProtocol implementation
+        """
+        # Lazy import dependencies only when class is instantiated
+        try:
+            from praisonaiui.datastore import BaseDataStore
+            # Make this class inherit from BaseDataStore at runtime
+            if BaseDataStore not in self.__class__.__bases__:
+                self.__class__.__bases__ = (BaseDataStore,) + self.__class__.__bases__
```

Ruff (0.15.11)

[warning] Consider `(BaseDataStore, *self.__class__.__bases__)` instead of concatenation

Replace with `(BaseDataStore, *self.__class__.__bases__)`

(RUF005)